### ISSUE TITLE
/dev/root at full storage capacity on sd card causing monerod and node to stop running

### ISSUE THREAD

**Issue by viperperidot** [monero-ecosystem/PiNode-XMR#10]
I noticed that /dev/root (on the pinodexmr web ui) which is mounted on / on my 16gb microsd card was slowly getting filled up. It eventually hit 100% which caused monerod to stop running. I looked inside / and it looks like the daemon.log file is stored in /var/log. This is only 1.6gb but if it continually grows then maybe that is the issue, I also noticed that there was 2gb of swap on the microsd card. The swap should be coming from the external ssd though, right? I tried disabling swap however this did not free up any space on /dev/root. Also when I look inside /dev I do not see the /root folder and /dev is only showing as having 12kb. On the hardware specs it says an 8gb sd card should be sufficient, so maybe something went wrong during my setup and files are getting written to the sd card that are not supposed to be? I am not exactly sure but I would like to get this sorted out so I can get monerod running again and continue with my initial sync. Is it ok to move the daemon.log to the ssd or will that cause pinodexmr to not work properly? Any help on getting my node running again would be appreciated. Basically I think we need to figure out why the sd card and /dev/root got filled up when my blockchain location is syncing to my external ssd, because I did follow all the instructions here on GitHub and I haven't heard of anyone else having this issue so I don't know what I did differently. Thanks

Edit: Should I try deleting the swap on the directory instead of using the webui, there was also a syslog that was fairly large file in the same folder as daemon.log but I think that file needs to stay right?

**Comment by viperperidot** [comment]
Update: So after doing some more research is appears that it is safe to delete log files in /var/log because they will just get created again. I looked at the tail of each log file and it appears that they are filling quickly because of the initial blockchain sync. However apparently there is supposed to be a process that raspbian runs to get rid of large log files before they become a problem but I guess that was not working. So I deleted 3 large log files in /var/log (daemon.log, syslog and syslog.1) and it appears that monerod is running again, I enabled the swap again but im still not sure whether that is coming from the sd card or ssd. I will leave it running for sync and then disable. Should I close this issue or would someone like to dig further and figure out why the log files are not getting automatically cleaned up before this problem happens?

**Comment by viperperidot** [comment]
Once the log files get regenerated I will see what is causing them to fill up so quickly.

**Comment by viperperidot** [comment]
Another update, we worked through this on Telegram but there was a hidden .bitmonero LMDB on the sd card with a bit of the blockchain on it. Somehow a small amount got written on there before switching to the SSD. We deleted this and it cleared up a few more GBs and now I am sitting at 78% on /dev/root which is much better. Will still monitor /dev/root in the future to see if the log files get out of control again. I suppose I can close this issue for now and re-open again if necessary
